### ISSUE TITLE
"Breakpoint in file that does not exist"

### ISSUE THREAD

**Issue by marcipops** [microsoft/vscode#144022]
Issue Type: <b>Bug</b>

Installed VS Code Community on Windows 10-Ubuntu 18.04
Ran VS Code by 'Code .' within Ubuntu folder that has primitive one line 'hello world' ipynb, to test.

Note all Python notebooks run fine within Ubuntu 
Also F10 Run Line by Line works fine.

However when using F5 'Start Debugging', it fails.
CALL STACK section says "NAME 'NULL' IS NOT DEFINED' and also breakpoints turns from red dot to hollow white dot and hovering over it states "Breakpoint in file that does not exist".
BREAKPOINTS section says "Breakpoint in file that does not exist" and shows a windows path name with backslashes rather than an Ubuntu path name with forward slashes.

I am new to VS Code, so if you need logs please instruct location.  Thanks in advance!

VS Code version: Code 1.64.2 (f80445acd5a3dadef24aa209168452a3d97cc326, 2022-02-09T22:02:28.252Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No
Remote OS version: Linux x64 5.10.60.1-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.81GB (6.60GB free)|
|Process Argv|--crash-reporter-id 323d3247-4b8e-4e48-beb9-06783ef35422|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-18.04|
|OS|Linux x64 5.10.60.1-microsoft-standard-WSL2|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1991)|
|Memory (System)|12.33GB (10.06GB free)|
|VM|0%|
</details><details><summary>Extensions (12)</summary>

Extension|Author (truncated)|Version
---|---|---
better-cpp-syntax|jef|1.15.13
jupyter-keymap|ms-|1.0.0
remote-containers|ms-|0.217.4
remote-ssh|ms-|0.74.0
remote-ssh-edit|ms-|0.74.0
remote-wsl|ms-|0.64.2
python-preview|don|0.0.4
python|ms-|2022.0.1814523869
vscode-pylance|ms-|2022.2.4
jupyter|ms-|2022.1.1301854968
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.6

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30443607
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30442553
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstartedc:30433508
vsclayoutctrc:30437038
vsrem710:30416614
dsvsc009:30440023
pythonvspyt640:30442238
vsbas813:30436447
vscscmwlcmc:30438804
helix:30440343
vsnot107cf:30443615

```

</details>

**Comment by marcipops** [comment]
Any advice would be welcome, thanks.

**Comment by weinand** [comment]
Please provide reproducible steps. Are you debugging Python in a VS Code notebook or just in file?

**Comment by marcipops** [comment]
Thanks Weinand.
Think I have fixed the issue using Ocamlfuse.  This python notebook is stored on Google Drive (as it originated from Colab).  So even though VS Code could access it to edit it on Windows (as it was mounted as a drive in Windows), when running the debugger, it could not resolve the path as Google Drive was not mounted in Ubuntu.
Now that I've installed Ocamlfuse and mounted the Google Drive, the paths are resolving properly, so I can use breakpoints now.
Many thanks
